css/index.css

```diff
@@ -33,22 +43,31 @@ a{
 /* - - - - - H E A D E R - - - - - */
 header{
 	color: black;
-	overflow: hidden;
 	position: relative;
-	margin-bottom: 20px;
+	gap: 1rem;
```

Comment:
The 'grid-gap' property must be in the `grid.css` file.

Reply:
Solved